pkgs/development/python-modules/scikit-learn/default.nix

```
@@ -54,17 +55,24 @@ buildPythonPackage rec {
     joblib
     threadpoolctl
   ];
-  checkInputs = [ pytest ];
+
+  checkInputs = [ pytest pytest-xdist ];
```

```suggestion
  checkInputs = [ pytestCheckHook pytest-xdist ];
```
You'll have to use `disabledTests` and `pytestFlagsArray`.

---

done